routes/conversation.js

```diff
@@ -169,6 +169,8 @@ exports.leave = function (req, res){
 			var participantIds = convo.participants.map(function (p){return p._id});
 			var userString = JSON.stringify(req.user._id);
 			var participantIdStrings = participantIds.map(function (id){return JSON.stringify(id)});
+			// Should this 'found' variable definition be changed to the equivalent 'foundIndex' variable
+			// definitions used below in 'exports.archive'?
```

Comment:
I can add that on to my 'leave functionality' PR, since it's a related bug.  Thanks for the discovery!